webcompat/form.py

```diff
@@ -428,15 +444,20 @@ def build_formdata(form_object):
     if extra_labels:
         metadata_keys.append('extra_labels')
 
+    clean = re.compile('<.*?>')
+    problem_type_stripped = re.sub(clean, '', get_radio_button_label(
+        form_object.get('problem_category'), problem_choices))
+    browser_tested_on = form_object.get('browser_test')
```

Comment:
The 3 lines here can be reduced with:
```suggestion
    browser_tested_on = form_object.get('browser_test', 'Unknown')
```